projects/ng-components-docs/src/app/examples/components/notification/notification.component.ts

```diff
@@ -0,0 +1,44 @@
+import { Component } from "@angular/core";
+import { NotificationAction } from "lib/src/notification";
+
+@Component({
+    selector: "app-notification",
+    templateUrl: "./notification.component.html",
+})
+export class NotificationComponent {
+    public notification1Toggle: boolean = false;
+    public notification2Toggle: boolean = false;
+    public notification3Toggle: boolean = false;
+
+    public description: string = "Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur.";
+    public notificationActions: Array<NotificationAction> = [
+        { text: "Yes, I'm in", action: () => this.onToggle2(false) },
+        { text: "Ignore", action: () => this.onToggle2(false) },
+    ];
+
+    onClick(e: MouseEvent) {
+        console.log("Notification clicked");
+    }
```

Comment:
is this necessary? No one will check the console while viewing the documentation